harvard-lts/CURIOSity
# Masonry Title Focus

**Masonry Title Focus**
* * *

**JIRA Ticket**: [LTSMAINT-765](https://jira.huit.harvard.edu/browse/LTSMAINT-765)

# What does this Pull Request do?
The item titles will now appear on focus when tabbing through images in masonry view (in addition to on hover like they currently do)

# How should this be tested?

A description of what steps someone could take to:
* Spin up Curiosity from this branch
* Navigate to any exhibit's search results and click on the masonry view
* Tabbing through the page, the item titles will now appear when in focus (previously only appeared on hover)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@VanessaVenti @enriquediaz
